### PR TITLE
LOG-4716: Add CI for logging vector 5.9

### PR DESCRIPTION
**Pull request by jcantrill**
https://issues.redhat.com/browse/LOG-4716

cc @syedriko @xperimental

### CODE REVIEW

File: ci-operator/config/ViaQ/vector/ViaQ-vector-release-5.8.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ base_images:
     namespace: logging
     tag: elasticsearch-proxy
   log-file-metric-exporter:
-    name: 5.y
+    name: "5.8"
```
Comment (xperimental) [inline ci-operator/config/ViaQ/vector/ViaQ-vector-release-5.8.yaml:31]: Does it make sense to keep this as `5.y` and build log-file-metric-exporter from master in our release?

**Comment by syedriko** [comment]
@jcantrill we already have https://github.com/openshift/release/pull/44618

**Comment by jcantrill** [comment]
> @jcantrill we already have https://github.com/openshift/release/pull/44618

We do but it is also missing the updates to the operator and other bits to properly test for 5.9

**Pull request merged**